src/Authenticated.js

```diff
@@ -0,0 +1,34 @@
+import React, { PropTypes } from 'react';
+import { AccountsClient } from '@accounts/accounts';
+import FormTypes from './FormTypes';
+
+class Authenticated extends React.Component {
+  static propTypes = {
```

Comment:
Yeah, because it is what we are using elsewhere. But if this is the style you prefer in this package, it is fine.